pycm/pycm_output.py

```diff
@@ -353,22 +353,32 @@ def sparse_table_print(sparse_matrix):
     return result
 
 
-def csv_matrix_print(classes, table):
+def csv_matrix_print(classes, table, header=False):
     """
     Return matrix as csv data.
 
     :param classes: classes list
     :type classes:list
     :param table: table
     :type table:dict
+    :param header: add headers to .csv file
+    :type header: bool
     :return:
     """
     result = ""
+    header_section = ""
     classes.sort()
     for i in classes:
+        header_section += str(i)
```

Comment:
In previous review I only mentioned second `if ... else` (adding`,` and `n`), the first `if` (creating header_section) position was true

Reply:
Alright, i fixed it.
There is a double check of `header` at the end of function due to it's right functionality for the cases that we do not need header, so last if (`if len(header_section > 0):`) is necessary i think.